src/migrate.py

```diff
@@ -74,10 +82,34 @@ def _migrate_import_params(settings_, import_file):
             elif isinstance(value, str):
                 settings.set_string(key, value)
             elif isinstance(value, int):
-                settings.set_int(key, value)
+                settings.set_int(key.replace("_", "-"), value)
     import_file.delete(None)
 
 
+def _migrate_styles(old_styles_dir, new_config_dir):
+    new_styles_dir = new_config_dir.get_child_for_display_name("styles")
+    if not new_styles_dir.query_exists(None):
+        new_styles_dir.make_directory_with_parents()
+    system_styles = styles.get_system_styles()
+    enumerator = old_styles_dir.enumerate_children("default::*", 0, None)
+    while 1:
```

Comment:
Personally I generally prefer while True, but that's not really a blocker to me.